<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="6cae3ea4-3a0d-493e-9c61-450402fd500c" name="Admin@D03 2017-10-10 14:19:31" runUser="D03\Admin" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="9d566280-59a1-456b-b459-cf21d28accde">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Admin_D03 2017-10-10 14_19_31">
      <DeploymentItem filename="C:\Users\Admin\Desktop\Calculator\BT_Calculator\bin\Debug\BT_Calculator.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2017-10-10T14:19:31.8415185+07:00" queuing="2017-10-10T14:19:31.8565194+07:00" start="2017-10-10T14:19:31.8675200+07:00" finish="2017-10-10T14:19:32.0715317+07:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TestMethod1" storage="c:\users\admin\desktop\calculator\caculatortester\bin\debug\caculatortester.dll" id="1488ab3d-b3c3-db00-47a5-4c2e50c79764">
      <Execution id="daec0724-844f-4313-816e-fa6a55e9a6cd" />
      <TestMethod codeBase="C:/Users/Admin/Desktop/Calculator/CaculatorTester/bin/Debug/CaculatorTester.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="CaculatorTester.UnitTest1, CaculatorTester, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TestMethod1" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="1488ab3d-b3c3-db00-47a5-4c2e50c79764" executionId="daec0724-844f-4313-816e-fa6a55e9a6cd" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="daec0724-844f-4313-816e-fa6a55e9a6cd" testId="1488ab3d-b3c3-db00-47a5-4c2e50c79764" testName="TestMethod1" computerName="D03" duration="00:00:00.0190547" startTime="2017-10-10T14:19:31.8805207+07:00" endTime="2017-10-10T14:19:32.0525306+07:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="daec0724-844f-4313-816e-fa6a55e9a6cd">
      <Output>
        <ErrorInfo>
          <Message>Test method CaculatorTester.UnitTest1.TestMethod1 did not throw expected exception System.DivideByZeroException.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>